Add component

tauren81 opened the pull request:
Не хватает еще последнего задания (тут же прикручен упаковщик для next.js)

tauren81 committed:
Проект vite от преподавателя для работы с компонентами

tauren81 committed:
Структура проекта

tauren81 committed:
Nextjs + component

tauren81 committed:
Создание своих компоент/ диалог без css

a4k commented on Nextjs/.next/types/package.json:
```diff
@@ -0,0 +1 @@
+{"type": "module"}
```
Папку .next лучше бы добавить в gitignore, чтобы не заполнял git

a4k reviewed (approved):
Approved! Молодец!

tauren81 committed:
Стили chacraui теперь работают и с tailwind. Добавил прлагин тайлвинд с Радиксом + стили модалки был добавлен импрот тейлвинд через стили темы, утилит, компонент. После этого стили подружились

@layer theme, base, components, utilities;

@import 'tailwindcss/theme.css' layer(theme);
@import 'tailwindcss/preflight.css' layer(base);
@import 'tailwindcss/utilities.css';

Так же предлагалось каждый тиль переименовать  префиксом для тайлвинд, чтобы с чакрой не было одинаковых названий (данный способ даже как-будто рабочий, но в кажый стиль пробрасывать префикс  - такое себе)

tauren81 committed:
Добавил исключения в гит для кеша nextjs
